### PR TITLE
show more detailed error message when importing/exporting

### PR DESCRIPTION
**Pull request by brianhatchl**
Old error popup:
Screenshot from 2019-06-20 14-39-58

New error popup:
Screenshot from 2019-06-20 14-40-33

### CODE REVIEW

File: modules/Hoot/ui/modals/exportData.js [hunk 1]
```diff
@@ -220,11 +220,19 @@ export default class ExportData {
             } )
             .catch( (err) => {
                 console.error(err);
+
                 let message = 'Error running export',
                     type = err.type,
                     keepOpen = true;
 
+                if (err.data.commandDetail.length > 0 && err.data.commandDetail[0].stderr !== '') {
+                    message = err.data.commandDetail[0].stderr;
```
Comment (bobi7) [inline modules/Hoot/ui/modals/exportData.js]: Do we want to overwrite the initial error message and not append to it instead?

File: modules/Hoot/ui/modals/importDataset.js [hunk 2]
```diff
@@ -383,10 +383,15 @@ export default class ImportDataset {
             .then( () => Hoot.events.emit( 'render-dataset-table' ) )
             .catch( err => {
                 console.error(err);
+
                 let message = 'Error running import',
                     type = err.type,
                     keepOpen = true;
 
+                if (err.data.commandDetail.length > 0 && err.data.commandDetail[0].stderr !== '') {
+                    message = err.data.commandDetail[0].stderr;
```
Comment (bobi7) [inline modules/Hoot/ui/modals/importDataset.js]: Same question as above

File: modules/Hoot/ui/modals/exportData.js (hunk 1 above)
Comment (brianhatchl) [reply modules/Hoot/ui/modals/exportData.js]: The only reason I did that for import multiple was that the user would want to know which file upload caused the error.  For single import and export I thought the error message from the hoot command was sufficient.

Comment (bobi7) [reply modules/Hoot/ui/modals/exportData.js]: good point

**Pull request merged**